fetch.bs

```diff
@@ -2230,6 +2230,9 @@ or "<code>object</code>".
 return true:
 
 <ol>
+ <li><p><a for=/>Assert</a>: <var>request</var>'s <a for=request>origin</a> is not
+ "<code>client</code>"..
```

Comment:
Perhaps my suggestion was wrong, but I notice this ends in two dots now. That's also the case in a few other places.